packages/react-native-worklets/src/shareables.ts

```diff
@@ -286,16 +286,11 @@ function cloneRemoteFunction<T extends object>(
 
 function cloneHostObject<T extends object>(
   value: T,
-  shouldPersistRemote: boolean
 ): ShareableRef<T> {
   // for host objects we pass the reference to the object as shareable and
   // then recreate new host object wrapping the same instance on the UI thread.
   // there is no point of iterating over keys as we do for regular objects.
-  const clone = WorkletsModule.makeShareableClone(
-    value,
-    shouldPersistRemote,
-    value
-  );
```

Comment:
On the Cpp side, we do not use these variables :
<img width="732" alt="image" src="https://github.com/user-attachments/assets/903d913e-880a-4999-9562-54b691f7f819" />